app/models/comment.rb

```diff
@@ -0,0 +1,23 @@
+class Comment < ActiveRecord::Base
+  belongs_to :commentable, :polymorphic => true
+  belongs_to :commenter, class_name: "User", foreign_key: :user_id
+
+end
+
+  def two_url(context)
```

Comment:
This could really be a view helper - give it the object and it would return the relevant path. It's not really a model concern what URL it's visible under.